swagger-api-client/swagger-client-build.txt

```diff
@@ -0,0 +1,21 @@
+lazy val root = (project in file(".")).
+  settings(
+    organization := "$organization",
+    name := "$name",
```

Comment:
I think you can get away without templating anything in this file. `organization` and `name` can just be set to broad and sam-client respectively. Things like version can be yanked from `sys.env` like we do for artifactory and snapshot-ness in workbench-libs. Etc.